### PR TITLE
BaSP-TG-61: Shared button component

### PR DESCRIPTION
**Pull request by melissabonillasellanes**
- Create a shared button component with two variants: primary and secondary
- Refactor in all the components where buttons are being used with the new props structure

### CODE REVIEW

**Review by matescarabino — APPROVED** [review]
Everything looks fine

**Review by Ansitox — APPROVED** [review]

File: src/Components/Shared/Button/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from './button.module.css';
+
+const Buttons = ({ type, variant, name, onClick }) => {
+  return (
+    <button type={type ? type : 'button'} className={`${styles[variant]}`} onClick={onClick}>
```
Comment (nicolasblasco) [inline src/Components/Shared/Button/index.js:6]: We should add disabled as a prop

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-router-dom": "^5.1.0",
+    "react-router-dom": "^5.3.4",
```
Comment (nicolasblasco) [inline package.json:11]: We should watch out if we change the version of react router dom

File: src/Components/Shared/Button/button.module.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.confirm {
```
Comment (nicolasblasco) [inline src/Components/Shared/Button/button.module.css:1]: Maybe we could rename this class to primary

File: src/Components/Shared/Button/button.module.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    background-color: #FF0000;
+    color: white;
+    font-size: 16px;
+    padding: 10px 30px;
+    text-align: center;
+}
+
+.cancel {
```
Comment (nicolasblasco) [inline src/Components/Shared/Button/button.module.css:9]: Maybe we could rename this class to secondary

**Review by nicolasblasco — CHANGES_REQUESTED** [review]

**Review by nicolasblasco — APPROVED** [review]

**Review by julianpiaggio — CHANGES_REQUESTED** [review]
🕵️‍♂️ QA FEEDBACK


First of all, we wanted to tell you that in order to approve this pr, some modifications need to be applied
that affect the functionality of the application. ⏰

This is:

* in the action of editing the superadmin, when wanting to confirm the update, it throws a banner without info and does not even return to the home screen. if we immediately press the confirm button again, it throws a banner with the legend "TypeError: Failed to fetch"

Then I leave you a link with a video of the error:
https://imgur.com/a/OEM4DU9


* in the creation of an employee when pressing the confirm button, it does not return to the initial page but stays on the same create screen

Then I leave you a link with a video of the error:
https://imgur.com/a/DZlAUtL

* in the update of employees when confirming the modification of an employee does not return to the initial page, but stays on the same screen.

Then I leave you a link with a video of the error:
https://imgur.com/a/AFjvgR1

**Review by julianpiaggio — APPROVED** [review]

**Pull request merged**